libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/ConfigureRouterTask.kt

```diff
@@ -2,18 +2,30 @@ package com.mapbox.services.android.navigation.v5.navigation
 
 import android.os.AsyncTask
 import com.mapbox.navigator.Navigator
+import com.mapbox.navigator.TileEndpointConfiguration
 
 internal class ConfigureRouterTask(
     private val navigator: Navigator,
     private val tilePath: String,
     private val callback: OnOfflineTilesConfiguredCallback
 ) : AsyncTask<Void, Void, Long>() {
 
-    @Synchronized override fun doInBackground(vararg paramsUnused: Void): Long =
-        navigator.configureRouter(tilePath)
+    @Synchronized
+    override fun doInBackground(vararg paramsUnused: Void): Long =
+        navigator.configureRouter(
+            tilePath,
+            null,
+            null,
+            TileEndpointConfiguration(
+                "https://api-routing-tiles-staging.tilestream.net",
```

Comment:
i agree we should use this endpoint as default for now and provide a setter to override it.

Reply:
Fixing.